presto-main/src/main/resources/webapp/src/components/ClusterHUD.jsx

```diff
@@ -181,10 +181,12 @@ export class ClusterHUD extends React.Component {
                     </div>
                     <div className="col-xs-4">
                         <div className="stat stat-large">
-                            <span className="stat-text">
-                                {this.state.activeWorkers[this.state.activeWorkers.length - 1]}
-                            </span>
-                            <span className="sparkline" id="active-workers-sparkline"><div className="loader">Loading ...</div></span>
+                            <a target="_blank" href="workers.html">
+                                <span className="stat-text">
```

Comment:
Let's add a glyphicon to the header instead (https://getbootstrap.com/docs/3.4/components/ glyphicon-list) and make that a link to make the color difference between the various primary stats less jarring.